llvm/lib/Target/SPIRV/SPIRVStructurizer.cpp

```diff
@@ -1218,7 +1218,7 @@ INITIALIZE_PASS_DEPENDENCY(DominatorTreeWrapperPass)
 INITIALIZE_PASS_DEPENDENCY(LoopInfoWrapperPass)
 INITIALIZE_PASS_DEPENDENCY(SPIRVConvergenceRegionAnalysisWrapperPass)
 
-INITIALIZE_PASS_END(SPIRVStructurizer, "structurize", "structurize SPIRV",
+INITIALIZE_PASS_END(SPIRVStructurizer, "structurizer", "structurize SPIRV",
```

Comment:
We should probably prefix the pass with `spirv-` and connect it the way the DXIL-specific passes are. That reduces the surface area of exposing target-specific passes in code that isn't target-specific.